CHANGELOG.rst

```diff
@@ -4,18 +4,30 @@ Changelog
 This document describes changes between each past release.
 
 
-0.3 (unreleased)
+1.0 (unreleased)
 ----------------
 
 **Breaking changes**
 
 - Most configuration entries were renamed, see `config/readinglist.ini`
   example to port your configuration.
+- status field was removed, archived and deleted fields were added
+- Remove Python 2.6 support
+
+**New features**
+
+- Add the /fxa-oauth/params endpoint
```

Comment:
nit: sometimes we use dots, sometimes ";", and sometimes nothing, I cannot find any general rule we're applying here. We should define one and follow it :)